### ISSUE TITLE
Docker for mac (native/xhyve)

### ISSUE THREAD

**Issue by rmoriz** [someara/kitchen-dokken#11]
Hi,

I'm running into the following error when using the new "native" Docker for mac:

```
-----> Starting Kitchen (v1.5.0)
-----> Creating <default-chef12-debian>...
       Finished creating <default-chef12-debian> (0m1.91s).
-----> Converging <default-chef12-debian>...
       Preparing files for transfer
       Preparing dna.json
       Resolving cookbook dependencies with Berkshelf 4.3.0...
       Removing non-cookbook files before transfer
       Preparing data_bags
       Preparing secret
       Preparing validation.pem
       Preparing client.rb
       Transferring files to <default-chef12-debian>
ssh: connect to host 127.0.0.1 port 32769: Connection refused
rsync: connection unexpectedly closed (0 bytes received so far) [sender]
rsync error: error in rsync protocol data stream (code 12) at /BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-47/rsync/io.c(453) [sender=2.6.9]
[2016-04-07T10:57:06+00:00] WARN: *****************************************
[2016-04-07T10:57:06+00:00] WARN: Did not find config file: /opt/kitchen/client.rb, using command line options.
[2016-04-07T10:57:06+00:00] WARN: *****************************************
[2016-04-07T10:57:06+00:00] WARN: No cookbooks directory found at or above current directory.  Assuming /.
[2016-04-07T10:57:06+00:00] FATAL: Cannot load configuration from /opt/kitchen/dna.json

```

.kitchen.yml

```yaml

---
driver:
  name: dokken
  chef_version: 12.5.1

transport:
  name: dokken

provisioner:
  name: dokken

platforms:
  - name: debian
    driver:
      vm_hostname: ci-deployer
      image: debian:jessie

verifier:
  root_path: '/opt/verifier'
  sudo: false

suites:
  - name: default-chef12
    run_list:
      - recipe[org-base]
      - recipe[org-deployer]
```

versions:

```

kitchen-dokken (0.0.29, 0.0.25)
kitchen-inspec (0.12.2)
kitchen-vagrant (0.19.0, 0.18.0)
test-kitchen (1.5.0, 1.4.2, 1.4.0)
```

docker output after the error:

````
docker ps
CONTAINER ID        IMAGE                          COMMAND                  CREATED             STATUS              PORTS                         NAMES
21a2f1544962        default-chef12-debian:latest   "sh -c 'trap exit 0 S"   9 minutes ago       Up 9 minutes                                      default-chef12-debian
ccfa1f30eb02        someara/kitchen-cache:latest   "/usr/sbin/sshd -D -p"   9 minutes ago       Up 9 minutes        192.168.64.10:32769->22/tcp   default-chef12-debian-data

```

**Comment by rmoriz** [comment]
Native docker provides a bonjour resource called "docker.local" which points to the IP that is used by the "native" virtual machine, in my case it's 192.168.64.10 but also uses the unix control socket on the host.

**Comment by rmoriz** [comment]
If you enable `VPN compatibility mode` in the Docker ui, all port mappings will be done to 127.0.0.1 (osx host in my case), this works.

**Comment by rmoriz** [comment]
with the current version it works quite nice:

```
Version 1.11.1-beta11 (build: 6974)
37559e5f6acd56a4810963acc7001e88f2d88017
```

**Comment by lamont-granquist** [comment]
% docker --version
Docker version 1.12.0-rc3, build 91e29e8, experimental

issue seems to be back, no pinata command, no advanced settings in the preferences to control this behavior, not sure how to go about fixing it...

**Comment by rmoriz** [comment]
FYI: "docker.local" and pinata are gone, they want to expose all relevant configs through the GUI.

**Comment by lamont-granquist** [comment]
yeah but i don't see it in the GUI either...

**Comment by someara** [comment]
is this still an issue with modern versions?

**Comment by rmoriz** [comment]
Works for me using the following versions:

- kitchen-dokken ref 4350872

- Docker for Mac: 
  Version 1.12.1-beta25 (build: 11807)
  158f2eefd23de930e87abae9c30ab3612b1e520a

**Comment by magicknot** [comment]
@someara I have this issue on Docker version 1.12.3, build 6b644ec

**Comment by someara** [comment]
@kevinreedy any ideas here?

**Comment by kevinreedy** [comment]
This should be fixed in https://github.com/someara/kitchen-dokken/pull/47, released as kitchen-dokken 1.0.9
